app/models/provider.rb

```diff
@@ -229,12 +229,8 @@ def update_changed_at(timestamp: Time.now.utc)
     update_columns changed_at: timestamp
   end
 
-  def unassigned_site_codes
-    Site::POSSIBLE_CODES - sites.pluck(:code)
-  end
-
   def can_add_more_sites?
```

Comment:
Could we just remove this altogether if it's always true?

Reply:
it is being used in the v2 provider serializer, should we move it over there instead?
https://github.com/DFE-Digital/teacher-training-api/blob/c455e1f36bb1f46172d89790b733e82a94e055b8/app/serializers/api/v2/serializable_provider.rb#L6

Reply:
Ok. We'll need to change publish to get rid of it in that case. Let's leave it for now and we'll do it in the tidy up.

Reply:
Carded up https://trello.com/c/EQEGSnMD/1664-remove-canaddmoresites-from-ttapi